### ISSUE TITLE
Play By Play not populating for MySql

### ISSUE THREAD

**Issue by mvgary094** [mpope9/nba-sql#102]
Hello,

I'm getting good data for all other tables but play_by_play isn't populating in my mysql db. I tried running just that table and lowering the limit for the number of rows needed to push but that didn't help. I've attached the output of the terminal and run configuration if that helps 

I'm also not seeing shot_chart_detail and player_general_traditional_total

![image](https://github.com/mpope9/nba-sql/assets/72046429/fcc3b3c2-af36-4a88-9bbf-0cd935cd329b)

![image](https://github.com/mpope9/nba-sql/assets/72046429/b1d5de02-e712-4214-ade2-de339d136f1e)

**Comment by mpope9** [comment]
Ah yeah, so in your command in the `--skip-tables` option you provide `pgtt` and `shot_chart_detail`. So that explains those two tables, however it doesn't describe the `play_by_play` issue. Are there any errors when running, or does it report a successful run?

**Comment by mvgary094** [comment]
Yea, it does run successfully. If you can see the rows[o] flags in the logs, you’ll find that it’s pulling good data. I’ll take another look after work, but I was starting to think it may be pushing to SQLite or something else instead

**Comment by mvgary094** [comment]
I was able to figure out that [the multithreading executor](https://github.com/mpope9/nba-sql/blob/0f557d2c946730986476a654f2bf69eca1a5b333/stats/nba_sql.py#L171) wasn't actually running the insert_batch command. I removed that just to see what happened and now I'm getting the following error:

peewee.IntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`nba`.`play_by_play`, CONSTRAINT `play_by_play_ibfk_3` FOREIGN KEY (`player1_id`) REFERENCES `player` (`player_id`))')

I then went in and manually dropped the foreign key constraints on the player_id fields and now I'm seeing data in the MySQL DB. 

Should I close this or should we try to figure out the root cause?
